packages/webworker/lumberjack/src/lib/logging/create-lumberjack.ts

```diff
@@ -16,10 +16,24 @@ interface LumberjackDependencies<TPayload extends LumberjackLogPayload | void> {
   getUnixEpochTicks: () => number;
 }
 
-export type Lumberjack<TPayload extends LumberjackLogPayload | void = void> = ReturnType<
-  typeof createLumberjack<TPayload>
->;
-
+/**
+ * Factory function that creates a Lumberjack core system object.
```

Comment:
nit: Use imperative tense.
```suggestion
* Create a Lumberjack core system object.
```